cwd/editor.js

```diff
@@ -96,16 +112,16 @@ views.forEach(view => {$.getJSON(view, function(data) {
 
         currentDiv.appendChild(document.createElement("BR"));
 
-        makeTextInput("input-" + viewItr + "-" + gaugeItr + "-" + messageItr, currentDiv, message.text);
+        makeTextInput("input-" + viewItr + "-" + gaugeItr + "-" + messageItr, currentDiv, message.text, null);
```

Comment:
No need to explicitly pass in `null` as an argument - if the argument is not passed, it seems that it will default to `undefined`, which is falsy anyhow.

Reply:
Is it necessary to change this? Or just less writing?

Reply:
This was more of an educational comment - I am resolving it because we are moving to React.